web/src/components/Layout.tsx

```diff
@@ -80,7 +80,7 @@ const Aside = styled.aside<{ asideStickyTop: number }>`
   display: inline-block;
   position: sticky;
   width: ${dimensions.toolbarWidth}px;
-  padding-top: max(35%, 80px);
+  top: 35%;
```

Comment:
This is a duplicate delcaration as the following is set above:
```
top: ${props => props.asideStickyTop}px;
```

Reply:
yea i updated the pr due to we have to ensure to have at least a top positioninf of (headerHeightMin+LanguageSelectionHeight)